ewasm-testnet-geth-config.json

```diff
@@ -7,6 +7,7 @@
     "eip155Block": 3,
     "eip158Block": 3,
     "byzantiumBlock": 4,
+    "ewasmBlock": 0,
```

Comment:
Hm, I don't think this should be before byzantium & co.

Reply:
It does not matter where it is. But I'd rather enable all from block 0.

Reply:
It does matter actually, because ewasm enforces byzantium rules in EEI :)